CameraXBasic/app/build.gradle

```diff
@@ -65,7 +68,7 @@ dependencies {
     implementation "androidx.navigation:navigation-ui-ktx:$nav_version"
 
     // CameraX core library
-    def camerax_version = '1.1.0-alpha04'
+    def camerax_version = '1.1.0-alpha07'
```

Comment:
I think there are stable versions now, so perhaps we should update to the stable versions?

Reply:
yeah, the stable one is 1.0.1, the latest one is 1.1.0-alpha04. Like to keep with the latest one in this repo.  in the talk about creating a "development" branch to host pre-release versions, not decided yet.

Reply:
got it, makes sense!